packages/reference/src/common/MultipleReferenceEditor.tsx

```diff
@@ -68,9 +68,11 @@ function Editor(props: EditorProps) {
     onLink: onLink,
   });
   const customCardRenderer = useCallback(
-    (cardProps: CustomEntryCardProps) =>
-      props.renderCustomCard ? props.renderCustomCard(cardProps, linkActionsProps) : false,
-    [linkActionsProps]
+    (cardProps: CustomEntryCardProps, linkActionsProps, renderDefaultCard) =>
+      props.renderCustomCard
+        ? props.renderCustomCard(cardProps, linkActionsProps, renderDefaultCard)
+        : false,
+    []
```

Comment:
@djagya We remove linkActionProps from here because they are passed to the customCardRenderer function everytime anyway. Is this ok with you?

Reply:
Hmm, where do they get passed from?

Reply:
If you mean https://github.com/contentful/field-editors/pull/430/files/3e343b599d9c8786e8e66cf93d32b4a5071f7b72#diff-f6ec592edf9c670a2aa37adfea260fb4R154, then it's just a placeholder so TS doesn't complain.

Actual `linkActionsProps` are generated in this place only.

Reply:
So no, it should stay as it was before, taking `linkActionsProps` from here: https://github.com/contentful/field-editors/pull/430/files/3e343b599d9c8786e8e66cf93d32b4a5071f7b72#diff-1eb231122762c9ef75c2fc235137581bR61

Reply:
Ok, thank you. I updated the PR. If ok we can merge